incubator/jaeger/templates/cassandra-schema-job.yaml

```diff
@@ -38,6 +38,30 @@ spec:
           value: {{ .Values.cassandra.config.dc_name | quote }}
         - name: CASSANDRA_PORT
           value: {{ .Values.storage.cassandra.port | quote }}
+        {{ if .Values.storage.cassandra.tls.enabled }}
+          - name: CASSANDRA_TLS
+            value: {{ .Values.storage.cassandra.tls.enabled }}
+          - name: CASSANDRA_TLS_SERVER_NAME
+            valueFrom:
+              secretKeyRef:
+                name: {{ .Values.storage.cassandra.tls.secretName }}
+                key: serverName
+          - name: CASSANDRA_TLS_KEY
+            valueFrom:
+              secretKeyRef:
+                name: {{ .Values.storage.cassandra.tls.secretName }}
+                key: key
+          - name: CASSANDRA_TLS_CERT
+            valueFrom:
+              secretKeyRef:
+                name: {{ .Values.storage.cassandra.tls.secretName }}
+                key: cert
+          - name: CASSANDRA_TLS_CA
+            valueFrom:
+              secretKeyRef:
+                name: {{ .Values.storage.cassandra.tls.secretName }}
+                key: ca
+          {{- end }}
```

Comment:
The key,cert and ca values need to be paths to the various files of the mounted secret volume, rather than the values directly according  to https://www.jaegertracing.io/docs/1.15/deployment/#tls-support

Unless  supplying the values directly works? Did you get a chance to test this @Pehesi97 ?

Reply:
In my opinion, it should be user's responsability to mount the files with extraConfigMapMounts and then use the paths in the secret. What do you think about it?

Reply:
This is my usage model.

![image](https://user-images.githubusercontent.com/8271749/70237409-e62b4880-1745-11ea-9887-42bd97997d5f.png)

Reply:
the `tlsSecret` would contain the actual values, not the paths, and if enabled, the chart should then mount the secret as a volume, and configure the paths in the various env vars. TLS cert data is better suited for a secret rather than a configMap

Reply:
Indeed. I'm making the changes right now.

Reply:
Done @naseemkullah. Could you please check my changes again?

Reply:
It's looking great, but could you also add the custom cqlshrc file to the schema job as per https://www.jaegertracing.io/docs/1.15/deployment/#tls-support?

Reply:
Done. @naseemkullah